k8s/upgrades/0.8.1-0.8.2/pre-check.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#####################################################################
+# NOTES: This script finds unlabeled volume resources of openebs   #
+#####################################################################
+
+# Search of CStor Resources
+printf "############## Unlabeled CStor Volumes Resources ##############\n\n"
+
+printf "CStor Volumes:\n"
+echo "--------------"
+printf "\n"
+# Search for CStor Volumes
+kubectl get cstorvolume --all-namespaces -l 'openebs.io/version notin (0.8.2), openebs.io/version notin (0.8.1)'
+
+printf "\nCStor Volumes Replicas:\n"
+echo "-----------------------"
+printf "\n"
+# Search for CStor Volume Replicas
+kubectl get cstorvolumereplicas --all-namespaces -l 'openebs.io/version notin (0.8.2), openebs.io/version notin (0.8.1)'
+
+printf "\nCStor Target service:\n"
+echo "---------------------"
+printf "\n"
+# Search for CStor Target Service
+kubectl get service --all-namespaces -l 'openebs.io/version notin (0.8.2), openebs.io/version notin (0.8.1), openebs.io/target-service in (cstor-target-svc)'
+
+printf "\nCStor Target Deployment:\n"
+echo "---------------------"
+printf "\n"
+# Search for CStor Target Service
```

Comment:
change the string as "Search for cStor Target deployment"

Reply:
Done